accounting/Cargo.toml

```diff
@@ -17,4 +17,4 @@ parity-scale-codec.workspace = true
 
 [dev-dependencies]
 proptest = "1.0"
-rstest = "0.16"
+rstest = "0.17"
```

Comment:
Perhaps this dependency can also be moved to the workspace level.

Reply:
Done